conf/application.conf

```diff
@@ -164,35 +164,66 @@ versioning {
 api-config {
 
   scopes {
-    "dummy-scope" {
-      fields = [ "A","B","C","D","E","F","G","H","I","J","K","L" ]
+    "read:individuals-benefits-and-credits-laa-c1" {
+        fields = ["B", "M", "D", "E", "F", "G", "N", "S", "H", "C", "O", "U", "I", "T", "P", "J", "K", "Q", "L", "R"]
     }
-    "read:hello-scopes-1" {
-      fields = [ "A","B","C"]
+    "read:individuals-benefits-and-credits-laa-c2" {
+        fields = ["B", "M", "D", "E", "F", "G", "N", "S", "H", "C", "O", "U", "I", "T", "P", "J", "K", "Q", "L", "R"]
     }
-    "read:hello-scopes-2" {
-      fields = [ "A","B"]
+    "read:individuals-benefits-and-credits-laa-c3" {
+        fields = ["B", "M", "D", "E", "F", "G", "N", "S", "H", "C", "O", "U", "I", "T", "P", "J", "K", "Q", "L", "R"]
+    }
+    "read:individuals-benefits-and-credits-hmcts-c2" {
+        fields = ["A", "B", "M", "D", "E", "F", "G", "N", "S", "H", "C", "O", "U", "I", "T", "P", "J", "K", "Q", "L", "R"]
+    }
+    "read:individuals-benefits-and-credits-hmcts-c3" {
+        fields = ["A", "B", "M", "D", "E", "F", "G", "N", "S", "H", "C", "O", "U", "I", "T", "P", "J", "K", "Q", "L", "R"]
+    }
+    "read:individuals-benefits-and-credits-lsani-c1" {
+        fields = ["A", "B", "M", "D", "E", "F", "G", "N", "S", "H", "C", "O", "U", "I", "T", "P", "J", "K", "Q", "L", "R"]
+    }
+    "read:individuals-benefits-and-credits-lsani-c3" {
+        fields = ["A", "B", "M", "D", "E", "F", "G", "N", "S", "H", "C", "O", "U", "I", "T", "P", "J", "K", "Q", "L", "R"]
     }
   }
 
   endpoints {
-    paye {
-      endpoint = "/individuals/employments/paye?matchId=$randomMatchId{&fromDate,toDate}"
-      fields {
-        A = "payments"
-        B = "employer/employerName"
-        C = "employer/employerAddress/line1"
-        D = "employer/employerAddress/line2"
-        E = "employer/employerAddress/line3"
-        F = "employer/employerDistrictNumber"
-        G = "employer/employerSchemeReference"
-        H = "employmentStartDate"
-        I = "employmentLeavingDate"
-        J = "employmentPayFrequency"
-        K = "employeeAddress"
-        L = "payrollId"
+    benefitsAndCredits {
```

Comment:
Also this endpoint config looks different to that in the test config used bu the code for testing the handler pattern. 

```
    endpoints = List(
      EndpointConfig(
        name = mockEndpoint1,
        link = "/a/b/c?matchId=<matchId>{&fromDate,toDate}",
        fields = Map(
          "A" -> "payments",
          "B" -> "employer/employerName=\"foo bar\"",
          "C" -> "employer/employerAddress/line1",
          "D" -> "employer/employerAddress/line2",
          "E" -> "employer/employerAddress/line3",
          "F" -> "employer/employerDistrictNumber",
          "G" -> "employer/employerSchemeReference",
          "H" -> "employmentStartDate",
          "I" -> "employmentLeavingDate",
          "J" -> "employmentPayFrequency",
          "K" -> "employeeAddress",
          "L" -> "payrollId"
        )
      ),
```

I think we will need to align to this name, link, fields 

Name is the string name of the endpoint used by the endpoint to fetch the scopes. 

Link is the full url for HAL 

Fields are the fields. 

@owen-opencast  can help confirm this?